proc_scripts/load_data.py

```diff
@@ -215,5 +215,7 @@ def proc_DOSY_CPMG(s):
         'spincore_IR_v1':proc_spincore_IR,
         'spincore_ODNP_v1':proc_spincore_ODNP_v1,
         'square_wave_capture_v1':proc_square_wave_capture,
+        'zg2h':proc_90_pulse,
+        'zg':proc_90_pulse,
         'DOSY_CPMG_v1':proc_DOSY_CPMG}
```

Comment:
I'm fixing a ROLLBACK ☹☹☹☹☹☹☹!

The idea here was that we could take advantage of the fact that Bruker pulse sequences will default to a preproc as given by the pulse sequence name (so if the pulse sequence is `zg2h`, you just give it this dictionary, and it would automatically know that it should use the `proc_90_pulse` routine)

Reply:
right but this pull request we aren't dealing with the proc_90_pulse, you said to make it a separate pull request, so I reverted this back to when proc_90_pulse wasn't included because we weren't dealing with that processing in this pull request

Reply:
touche!

Reply:
I just pushed a commit where I pull it out again